module/epochs/qc_voter.go

```diff
@@ -124,7 +125,8 @@ func (voter *RootQCVoter) Vote(ctx context.Context, epoch protocol.Epoch) error
 		// submit the vote - this call will block until the transaction has
 		// either succeeded or we are able to retry
 		log.Info().Msg("submitting vote...")
-		err = voter.client.SubmitVote(ctx, vote)
+		voteMessage := hotstuffver.MakeVoteMessage(vote.View, vote.BlockID)
+		err = voter.client.SubmitVote(ctx, vote, voteMessage)
```

Comment:
I don't think we need to change the interface here.

The `RootQCVoter` is already passing the `model.Vote` object (which contains `View` and `BlockID`) to the `QCClient`. The `QCClient` can generate the vote message on its own.
```suggestion
		err = voter.client.SubmitVote(ctx, vote)
```